test/oracle/helpers/callback.mligo

```diff
@@ -0,0 +1,38 @@
+#import "../../../src/contracts/cameligo/oracle/callback/main.mligo" "Callback"
+
+(* Some types for readability *)
+type taddr = (Callback.parameter, Callback.storage) typed_address
+type contr = Callback.parameter contract
+type originated = {
+    addr: address;
+    taddr: taddr;
+    contr: contr;
+}
+
+let plain_timestamp : timestamp = ("1970-01-01T00:00:01Z" : timestamp)
+
+(* Base Callback storage *)
+let base_storage : Callback.storage = {
+    name = "";
+    videogame = "";
+    begin_at = plain_timestamp + 2000;
+    end_at = plain_timestamp + 4000;
+    modified_at = plain_timestamp;
+    opponents = { team_one = ""; team_two = ""};
+    is_finalized = false;
+    is_draw = (None : bool option);
+    is_team_one_win = (None : bool option);
+    metadata = (Map.empty : (string, bytes) map);
+}
+
+let originate_from_file (initial_storage : Callback.storage) : originated =
+    let oracle_path = "src/contracts/cameligo/oracle/callback/main.mligo" in
```

Comment:
same as previous feedback related to relative paths should work with ligo version `0.51.0`

Reply:
Thanks